.github/workflows/update-terraform-provider.yaml

```diff
@@ -0,0 +1,70 @@
+name: Update Terraform Provider
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch: {}
+
+jobs:
+  update-terraform-provider:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Check for New Terraform Provider Release
+        id: check_release
+        run: |
+          latest_release=$(gh release list -R scaleway/terraform-provider-scaleway --limit 1)
```

Comment:
Could all of this be in a script that we could also launch locally? I'm concerned about putting too much logic inside the github action.